src/test/index.ts

```diff
@@ -2,7 +2,32 @@ import * as path from 'path';
 import * as Mocha from 'mocha';
 import * as glob from 'glob';
 
+function setupNyc() {
+	const NYC = require('nyc');
+	const defaultExclude = require('@istanbuljs/schema/default-exclude');
+	const cwd = path.join(__dirname, '..', '..', '..');
+	const nyc = new NYC({
+	  cache: true,
+	  cacheDir: path.join(cwd, '.cache', 'nyc'),
+	  cwd,
+	  exclude: [...defaultExclude, '**/.vscode-test/**'],
+	  extensions: ['ts'],
+	  hookRequire: true,
+	  hookRunInContext: true,
+	  hookRunInThisContext: true,
+	  instrument: true,
+	  reporter: ['text', 'html'],
+	  sourceMap: true,
+	  tempDirectory: path.join(cwd, 'coverage', 'integration')
```

Comment:
[F] This too, we have a jest test suite that runs so we need to run multiple coverage reports into different folders and then merge them. I think (not sure) that you might be able to drop this line.